dolphinscheduler-task-plugin/dolphinscheduler-task-jupyter/src/main/java/org/apache/dolphinscheduler/plugin/task/jupyter/JupyterTask.java

```diff
@@ -220,7 +200,6 @@ private List<String> populateJupyterOptions() {
 
     @Override
     public void cancelApplication(boolean cancelApplication) throws Exception {
-        // cancel process
         shellCommandExecutor.cancelApplication();
```

Comment:
Currently, jupyter task runs only on workers. Jupyter task plugin uses shell command `papermill xxxx` to evaluate jupyter notes. If shell process cancelled, `papermill` get canceled, which means jupyter task is cancelled. `papermill` does not rely on an external jupyter server. It could run jupyter notes independently.

Reply:
@caishunfeng BTW, if we could support `Shell on Yarn` task in the future, we could make jupyter task running in the yarn containers of remote machines.